compiler/rustc_trait_selection/src/traits/select/confirmation.rs

```diff
@@ -600,7 +600,10 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
         debug!(?obligation, "confirm_fn_pointer_candidate");
 
         // Okay to skip binder; it is reintroduced below.
```

Comment:
This comment is now wrong, there's no binder being skipped

Reply:
Good point.